react/react_events/README.md

```diff
@@ -77,7 +79,7 @@ Alright, let's look at this line-by-line. First, we're using object destructurin
 
 Then, we return our JSX. We've got a containing `div`, because our component can only render one element at a time. Inside that `div`, we have a `button` element with an **event listener**: `onClick`.
 
-`onClick` is one example of how JSX listens for certain events. For the most part, it does what it says - it triggers when the element is clicked. However, take a look at those curly braces (`onClick={this.handleClick}`)! These braces, unique to JSX, allow us to insert JavaScript code into our HTML. We can do this for a few different purposes. In this case, we're doing it to specify what should happen when we click the button. `onClick` **listens** for an event, whereas `handleClick` **handles** what happens afterwards - our `handleClick` function is therefore known as an **event handler**.
+`onClick` is one example of how JSX listens for certain events. For the most part, it does what it says - it triggers when the element is clicked. However, take a look at those curly braces (`onClick={this.handleClick}`)! These curly braces, unique to JSX, allow us to insert JavaScript code into our HTML. We can do this for a few different purposes. In this case, we're doing it to specify what should happen when we click the button. `onClick` **listens** for an event, whereas `handleClick` **handles** what happens afterwards - our `handleClick` function is therefore known as an **event handler**.
```

Comment:
Comment on line 78. That paragraph mentions object destructuring. Have we taught or introduced that yet?

Reply:
We haven't introduced that,  should we keep the terminology or explain what that means ?